controlpanel/frontend/views/user.py

```diff
@@ -29,6 +29,13 @@ class UserList(LoginRequiredMixin, PermissionRequiredMixin, ListView):
     queryset = User.objects.exclude(auth0_id='')
     template_name = "user-list.html"
 
+    def get_ordering(self):
+        order_by = self.request.GET.get("o", "username")
```

Comment:
Is a Django convention to call this ordering query string param just `o`? Or we could change to something more explicit like `order_by` for example? (basically same as you named the variable).

Reply:
(TBH from screenie, it already seems to be sorted alphabetically by `username` so that's good)

Reply:
So this is the golden nugget of information: yes, it's the convention (but finding out the conventions often involves knowing what to look for / ask). When I'd done this before I'd either used a JS library (which I wanted to avoid) or done it with response functions (rather than classes).

Every model can have a default ordering IIRC. For users I believe that's username, otherwise it's PK if not set. Since we inherit from `django.contrib.auth.User` then the default was sort-by-username.

The beauty is we can override this in such a small amount of code. :-)